projects/bp-gallery/src/components/common/PictureOverview.tsx

```diff
@@ -32,10 +34,14 @@ const PictureOverview = ({
     false,
     sortBy,
     true,
-    ABSOLUTE_MAX_PICTURES_PER_ROW * rows
+    ABSOLUTE_MAX_PICTURES_PER_ROW * rows,
+    'cache-and-network',
```

Comment:
needed?

Reply:
Yes. Actually we would have to use 'no-cache' to avoid a very unlikely edge case.